packages/react-badge/src/Badge/useBadgeStyles.ts

```diff
@@ -0,0 +1,110 @@
+import { makeStyles, ax } from '@fluentui/react-make-styles';
+import { BadgeState } from './Badge.types';
+
+/**
+ * Styles for the root slot
+ */
+export const useRootStyles = makeStyles<BadgeState>([
+  [
+    null,
+    theme => ({
+      display: 'inline-flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      backgroundColor: theme.alias.color.brand.brandBackground,
+      borderColor: theme.alias.color.brand.brandBackground,
+      color: theme.global.color.white,
+    }),
+  ],
+  [
+    s => s.size === 'smallest',
+    theme => ({
+      width: '6px',
+      height: '6px',
+      fontSize: '4px',
+    }),
+  ],
+  [
+    s => s.size === 'smaller',
+    theme => ({
+      width: '10px',
+      height: '10px',
+      fontSize: '6px',
+    }),
+  ],
+  [
+    s => s.size === 'small',
+    theme => ({
+      minWidth: '4px',
+      height: '16px',
+      paddingRight: '6px',
+      paddingLeft: '6px',
+      gap: '4px',
+      fontSize: '8px',
+    }),
+  ],
+  [
+    s => s.size === 'medium',
+    theme => ({
+      height: '20px',
+      minWidth: '4px',
+      gap: '4px',
+      paddingRight: '8px',
+      paddingLeft: '8px',
+      fontSize: '12px',
+    }),
+  ],
+  [
+    s => s.size === 'large',
+    theme => ({
+      minWidth: '8px',
+      height: '24px',
+      paddingRight: '8px',
+      paddingLeft: '8px',
+      fontSize: '12px',
+      gap: '6px',
+    }),
+  ],
+  [
+    s => s.size === 'larger' || s.size === 'largest',
+    theme => ({
+      minWidth: '8px',
+      height: '32px',
+      paddingRight: '12px',
+      paddingLeft: '12px',
+      gap: '6px',
+      fontSize: '12px',
+      border: '2px',
+    }),
+  ],
+  [s => s.shape === 'circular', theme => ({ borderRadius: theme.global.borderRadius.circular })],
+  [s => s.shape === 'rounded', theme => ({ borderRadius: theme.global.borderRadius.medium })],
+  [
+    s => s.shape === 'rounded' && (s.size === 'small' || s.size === 'smaller' || s.size === 'smallest'),
+    theme => ({ borderRadius: theme.global.borderRadius.small }),
+  ],
+]);
+
+/**
+ * Styles for the icon slot
+ */
+export const useIconStyles = makeStyles<BadgeState>([
+  [
+    s => !s.children,
+    () => ({
+      position: 'absolute',
+    }),
+  ],
+]);
+
+/** Applies style classnames to slots */
```

Comment:
I rectified my mistake one line jsdoc comments in subsequent PRs, let's follow standard... sorry !
```suggestion
/** 
* Applies style classnames to slots
*/
```